Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cargo-ledger"
-version = "1.0.0"
+version = "1.0.1"
```

Comment:
Maybe this should be at least a minor bump and not just a patch bump as this introduces a new important feature?

Reply:
agreed. updated